IdentityCore/tests/MSIDBrowserNativeMessageGetTokenRequestTests.m

```diff
@@ -71,6 +71,7 @@ - (void)testInitWithJSONDictionary_whenJsonValidAndAllFieldsProvided_shouldInit
             @"correlationId": @"9BBCA391-33A9-4EC9-A00E-A0FBFA71013D",
             @"prompt": @"login",
             @"isSts": @(YES),
+            @"canShowUI": @(NO),
```

Comment:
nit: in code you assume it's NSString, but tests use NSNumber. I'd suggest to keep this consistent.

Reply:
good point, we support both during conversion, updated the code to not assume it is a string.